plugin-ci/orb.yml

```diff
@@ -24,19 +24,36 @@ commands:
           from: << parameters.filename >>
           to: << parameters.bucket >>
           arguments: '--acl public-read --cache-control no-cache'
+  install-node:
+    description: Install node
+    steps:
+      - run:
+          name: Install Node
+          command: |
+            set +e
+            wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.1/install.sh | bash
+            export NVM_DIR="$HOME/.nvm"
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"
+
+            nvm install || nvm install 13.14 # use `.nvmrc` if it exists, otherwise use 13.14
```

Comment:
What's actually happening on this line:

If `.nvmrc` exists, `nvm` will use that version when running `nvm install` with no arguments. This command will return a non-zero error code if `.nvmrc` doesn't exist or something fails. We then fall back to `13.14` in this case. THe majority of plugin projects are on npm 6 at the moment so this is for backwards compatibility. All a plugin project needs to do to use a more recent version of npm is to create a `.nvmrc` file.

Reply:
Should this functionality be documented? E.g. in the orb description?